jenkins.sh

```diff
@@ -52,7 +52,7 @@ if [[ ${GIT_BRANCH} != "origin/master" ]]; then
     app spec lib spec config
 fi
 
-GOVUK_CONTENT_SCHEMAS_PATH=tmp/govuk-content-schemas RAILS_ENV=test bundle exec rake
+GOVUK_CONTENT_SCHEMAS_PATH=tmp/govuk-content-schemas RAILS_ENV=test xvfb-run -a bundle exec rake
```

Comment:
Does anyone else have the same problem on the dev vm?  Maybe it's just my setup? 

FWIW: every other app that does this uses Poltergeist - perhaps because of this very issue?